Rakefile

```diff
@@ -51,4 +51,6 @@ task 'hoe:spec' do
   File.open("#{hoe.name}.gemspec", "w") { |f| f.write hoe.spec.to_ruby}
 end
 
+desc 'verify code quality before committing changes'
+task 'verify' => ['clean', 'test', 'rubocop', 'bundler:gemfile', 'git:manifest']
```

Comment:
`git:manifest` would generate the Manifest.txt file. Do we need to have this file then in the repository at all? Could we generate it just before we do `rake package` maybe?

Reply:
Hm...Good idea. But there are some hoe tasks except `rake package` that may require manifest.txt, so I think it’s good to keep it all the time.